From 2ec8258fe4d039ce312a1e59ba4b8419ad96d808 Mon Sep 17 00:00:00 2001
From: Georgi Djakov <georgi.djakov@linaro.org>
Date: Thu, 17 Nov 2016 17:20:26 +0200
Subject: ARM: qcom_defconfig: Enable RPM/RPM-SMD clocks

From: Georgi Djakov <georgi.djakov@linaro.org>

commit 2ec8258fe4d039ce312a1e59ba4b8419ad96d808 upstream.

Enable support for clocks, controlled by the RPM processor on
Qualcomm platforms.

Signed-off-by: Georgi Djakov <georgi.djakov@linaro.org>
Signed-off-by: Andy Gross <andy.gross@linaro.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

---
 arch/arm/configs/qcom_defconfig |    2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm/configs/qcom_defconfig
+++ b/arch/arm/configs/qcom_defconfig
@@ -157,6 +157,8 @@ CONFIG_DMADEVICES=y
 CONFIG_QCOM_BAM_DMA=y
 CONFIG_STAGING=y
 CONFIG_COMMON_CLK_QCOM=y
+CONFIG_QCOM_CLK_RPM=y
+CONFIG_QCOM_CLK_SMD_RPM=y
 CONFIG_APQ_MMCC_8084=y
 CONFIG_IPQ_LCC_806X=y
 CONFIG_MSM_GCC_8660=y
